addon/components/sortable-group.js

```diff
@@ -111,7 +118,11 @@ export default Component.extend({
         items.invoke('thaw');
       });
     });
-
-    this.sendAction('onChange', models);
+    
+    if (groupModel === null) {
```

Comment:
My fear is that a `null` model may arrive due to an unrelated error but it’ll be unclear what’s gone wrong when it gets to the action handler because rather than a `null` argument in the first position you’ll get a list of items.

The [suggestion above](https://github.com/jgwhite/ember-sortable/pull/28#issuecomment-102829337) uses a unique sentinal value:

```js
const NO_MODEL = {};
```

This is guaranteed not to collide with any other value `model` might have.